src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -18,31 +28,98 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
   display: grid;
 `;
 
+const AmpIframe = ({ children, className, width, height, src }) => (
+  <amp-iframe
+    class={className}
+    width={width}
+    height={height}
+    layout="responsive"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation-by-user-activation allow-forms"
+    resizable
+    src={src}
+  >
+    {children}
+  </amp-iframe>
+);
+
+const StyledAmpIframe = styled(AmpIframe)`
+  > [overflow] {
+    background: linear-gradient(
+      0deg,
+      #fff 0%,
+      rgba(255, 255, 255, 1) 75%,
+      rgba(255, 255, 255, 0) 100%
+    );
+    display: flex;
+    justify-content: center;
+
+    &::after {
+      background-color: #fff;
+      border-top: ${KEYLINE_WIDTH} solid ${C_EBON};
+      content: '';
+      display: block;
+      height: 50%;
+      left: 0;
+      position: absolute;
+      top: 50%;
+      width: 100%;
+      z-index: -10;
```

Comment:
Where does the `z-index` `-10` value come from, I guess it makes the`show more' button 'low priority', what is it competing with?

Reply:
This rule is added to the pseudo-element we create with `::after`, which creates the 'keyline'. It ensures it's always _behind_ the button.
It's not competing with anything as such - I tend to [follow the practice](https://tailwindcss.com/docs/z-index) of rounding z-indexes to multiples of 10 as it gives you a bit of breathing room and they're easier to keep track of.